pom.xml

```diff
@@ -19,7 +19,7 @@
     <url>https://craftaro.com/marketplace/product/13</url>
 
     <properties>
-        <craftaro.coreVersion>3.1.0-SNAPSHOT</craftaro.coreVersion>
+        <craftaro.coreVersion>3.3.0-SNAPSHOT</craftaro.coreVersion>
```

Comment:
I'm a bit busy this week – Was core v3.2 the version where ServerVersion and/or Mini-Message changed making some migration steps required?
I'm not sure right now

Reply:
I merge it and I'll fix anything else if needed.